src/picongpu/include/plugins/ChargeConservation.tpp

```diff
@@ -69,7 +69,7 @@ void ChargeConservation::pluginLoad()
     zone::SphericZone<simDim> zone_allGPUs(gpuDim);
     this->allGPU_reduce = AllGPU_reduce(new PMacc::algorithm::mpi::Reduce<simDim>(zone_allGPUs));
 
-    if(this->allGPU_reduce->root())
+    if(this->allGPU_reduce->root() && this->notifyPeriod > 0u)
```

Comment:
please move this to the top of the method to avoid any action of this plugin if it is not needed. (e.g. set `setNotificationPeriod()`)